network/ClientNetworking.cpp

```diff
@@ -285,6 +292,10 @@ bool ClientNetworking::Impl::PlayerIsHost(int player_id) const {
     return player_id == m_host_player_id;
 }
 
+bool ClientNetworking::Impl::HasAuthRole(Networking::RoleType role) const {
+    return m_roles.HasRole(role);
```

Comment:
for single line functions, prefer `{ /* stuff */ }` all on one line below signature line.

Reply:
Fixed.